tides-server/pkg/handler/user_api.go

```diff
@@ -10,7 +12,45 @@ import (
 
 	"tides-server/pkg/config"
 	"tides-server/pkg/models"
+
+	"crypto/tls"
+	gomail "gopkg.in/mail.v2"
+	"github.com/sethvargo/go-password/password"
 )
+const OFFICIAL_EMAIL = "cloudtest2021abc@gmail.com"
+const OFFICIAL_PASSWORD = "cloudtides"
+
+// ResetPasswordHandler is the API for /users/reset POST
+func ResetPasswordHandler(params user.ResetPasswordParams) middleware.Responder {
+  fmt.Println("entered!")
+  body := params.ReqBody
+  db := config.GetDB()
+  var u models.User
+  fmt.Println(body.Username + ", " + body.Password + ", " + body.NewPassword)
```

Comment:
don't print out the username and password, if this is for test, please delete it

Reply:
@yvvym to resolve the issue